aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client/sample/client/Program.cs

```diff
@@ -42,38 +52,55 @@ static void ShowProduct(Product product)
             }
             return product;
         }
+        #endregion
 
+        #region snippet_UpdateProductAsync
         static async Task<Product> UpdateProductAsync(Product product)
         {
-            HttpResponseMessage response = await client.PutAsJsonAsync($"api/products/{product.Id}", product);
+            HttpResponseMessage response = await client.PutAsJsonAsync(
+                $"api/products/{product.Id}", product);
             response.EnsureSuccessStatusCode();
 
             // Deserialize the updated product from the response body.
             product = await response.Content.ReadAsAsync<Product>();
             return product;
         }
+        #endregion
 
+        #region snippet_DeleteProductAsync
         static async Task<HttpStatusCode> DeleteProductAsync(string id)
         {
-            HttpResponseMessage response = await client.DeleteAsync($"api/products/{id}");
+            HttpResponseMessage response = await client.DeleteAsync(
+                $"api/products/{id}");
             return response.StatusCode;
         }
+        #endregion
 
         static void Main()
         {
             RunAsync().Wait();
         }
 
+        #region snippet_run
+        #region snippet5
         static async Task RunAsync()
         {
-            client.BaseAddress = new Uri("http://localhost:55268/");
+            Console.WriteLine("Update port # in the following line.");
```

Comment:
Why a Console.Writeline? If it's instructing the reader, shouldn't it be outside the code block or a comment? Because it looks like they should actively update the port through the 
 cmd line console app itself. This is somewhat unclear.